manageiq-ui-classic.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.executables   = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency "rails", "~> 6.0.0"
+  s.add_dependency "rails", ">= 6.0.4", "< 7.0"
```

Comment:
Previously this would allow 6.0.0, but 6.0.4 is now the minimum?

Reply:
Yeah, that was intentional for master since ui-classic is tightly coupled to ManageIQ and the above requirement is closer to the [app's Gemfile](https://github.com/ManageIQ/manageiq/blob/2d1d489c02009e35b3d94f3f50d05cf419dbd210/Gemfile#L67).  6.0.0 was released August 16, 2019. 6.0.4 was released  June 15, 2021.

I can do that separately if we want.

Reply:
That's fine, just wanted to make sure it was intentional

Reply:
I don't think we need this line at all.  Can you try removing it completely?  For example, manageiq-api nor any of the other plugins have it.

Reply:
@jrafanie ... free bonus commit that's a delete 😉

Reply:
> I don't think we need this line at all. Can you try removing it completely? For example, manageiq-api nor any of the other plugins have it (except schema, but that one is special because it tests within a dummy app)

If we keep ui-classic separate from core, it should have at least similar dependencies of the core repo but maybe less strict.  I don't see the benefit of completely dropping a dependency if the code here actually won't work with other versions.

Reply:
I think @Fryguy is saying that the UI will actually not work without the core repo, as it depends on models defined there.

Reply:
Right...plugins can't work without core, and core defines the rails version.  Having it duplicated is unnecessary.

Reply:
Thanks, I can see that.  I really wish manageiq could be an explicit dependency in the gemspec here or at least make this coupling more obvious.  I can imagine it's easy to checkout lasker of a plugin and forget to switch off of master of manageiq.

But, because we're using the gem specification here though, I think a minimum rails version here is fine and core/manageiq can drive the actual version requirement.  I'm 💯 for making this less painful for upgrades of future rails versions.